app/store/models/Registration.d.ts

```diff
@@ -57,7 +62,7 @@ export type PaymentRegistration = Pick<Registration, 'paymentStatus'> &
 export type DetailedRegistration = Pick<
   Registration,
   | 'id'
-  | 'user'
```

Comment:
What is this? There is no `detailedUser` field on registrations??

Reply:
I should be passing it from the backend

```ts
interface Registration {
  id: ID;
  user: PublicUser;
  detailedUser: DetailedUser; // Has just been added to the interface here
  createdBy: ID;
	//...
}
```\